shared/teams/team/tabs/index.stories.js

```diff
@@ -0,0 +1,54 @@
+// @flow
+import * as React from 'react'
+import {action, storiesOf} from '../../../stories/storybook'
+import TeamTabs from '.'
+
+const yourOperations = {
+  manageMembers: false,
+  manageSubteams: false,
+  createChannel: false,
+  chat: false,
+  deleteChannel: false,
+  renameChannel: false,
+  editChannelDescription: false,
+  setTeamShowcase: false,
+  setMemberShowcase: false,
+  setRetentionPolicy: false,
+  changeOpenTeam: false,
+  leaveTeam: false,
+  joinTeam: false,
+  setPublicityAny: false,
+  listFirst: false,
+  changeTarsDisabled: false,
+  deleteChatHistory: false,
+}
+
+const commonProps = {
+  admin: false,
+  loading: false,
+  memberCount: 12,
+  newTeamRequests: ['Cool Team 😎'],
+  numInvites: 5,
+  numRequests: 0,
+  numSubteams: 0,
+  resetUserCount: 0,
+  selectedTab: 'members',
+  setSelectedTab: action('setSelectedTab'),
+  teamname: 'Cool Team 😎',
+  yourOperations,
+}
+
+const load = () => {
+  storiesOf('Teams/Tabs', module)
+    .add('Standard', () => <TeamTabs {...commonProps} />)
+    .add('Loading', () => <TeamTabs {...commonProps} loading={true} />)
+    .add('Admin', () => <TeamTabs {...commonProps} admin={true} />)
+    .add('User can Manage Subteams', () => (
+      <TeamTabs {...commonProps} yourOperations={{...yourOperations, manageSubteams: true}} />
+    ))
+    .add('Has Subteams', () => <TeamTabs {...commonProps} numSubteams={3} />)
+    .add('Has reset users', () => <TeamTabs {...commonProps} resetUserCount={7} />)
+    .add('Has team requests', () => <TeamTabs {...commonProps} admin={true} numRequests={4} />)
```

Comment:
This and `Has reset users` look a little funky:
<img width="124" alt="image" src="https://user-images.githubusercontent.com/11968340/42908984-78280116-8ab0-11e8-9861-f79b7eba5545.png">
<img width="102" alt="image" src="https://user-images.githubusercontent.com/11968340/42908987-7ce6f950-8ab0-11e8-89e9-140b99993f48.png">

Reply:
Oh, this is how it looks on master currently

Reply:
Maybe #12798 is the cause? (cc @adamjspooner) I wrote this story a while ago, and I remember it looking fine then

Reply:
Fixed by https://github.com/keybase/client/commit/f57fcc4d5ed8ff4dede1977c6800ced7c1230a22